src/generate/CodeModelAzImpl.ts

```diff
@@ -2269,10 +2360,27 @@ export class CodeModelCliImpl implements CodeModelAz {
         return extensions && 'x-ms-examples' in extensions ? extensions['x-ms-examples'] : {};
     }
 
+    public get Example(): unknown {
+        return this.Examples[this.currentExampleIndex];
+    }
+
     public get ExampleAmount(): number {
         return Object.keys(this.Examples).length;
     }
 
+    public get AzExample(): CommandExample {
+        return this.Method_AzExamples[this.currentAzExampleIndex];
+    }
+
+    public get AzExample_CommandString(): string {
+        return this.AzExample.CommandString;
+    }
+
+    public get AzExample_CommandStringItems(): string[] {
+        const items = [];
+        ToMultiLine(this.AzExample_CommandString, items, 119, true);
```

Comment:
what does '119' mean?  A magic number?

Reply:
That's pylint max code length, let me change it into constant.

Reply:
changed, and I also changed all the file that referenced pylint max code length 119.